account_invoice_inter_company/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2013-2014 Odoo SA
+# Copyright 2015-2017 Chafique Delli <chafique.delli@akretion.com>
+# Copyright 2020 Tecnativa - David Vidal
+# Copyright 2020 Tecnativa - Pedro M. Baeza
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Inter Company Invoices",
+    "summary": "Intercompany invoice rules",
+    "version": "16.0.1.0.0",
+    "category": "Accounting & Finance",
+    "website": "https://github.com/OCA/multi-company",
+    "author": "Odoo SA, Akretion, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "depends": ["account", "base_setup"],
+    "data": [
+        "data/inter_company_data.xml",
```

Comment:
I think it s demo, not data.

Reply:
@Kev-Roche , @hbrunn , Indeed, I had asked myself the question.

Reply:
@Kev-Roche , @hbrunn , it's modified.